### PR TITLE
Add Meta Tags, Favicon, OpenGraph Move CSS #34

### PR DESCRIPTION
**Pull request by RyanJField**
I have added the SEO Tags per (#34)

I have also moved the CSS to a separate file, and added a favicon.

### CODE REVIEW

File: ui.R [hunk 1]
```diff
@@ -1,794 +1,797 @@
...
+  dashboardBody(
+  tags$head(
+    # load custom stylesheet
+    # To ensure white background and no horizontal scroll bars on ranking panel
+    tags$link(rel = "stylesheet", type = "text/css", href = "app.css"),
+    includeHTML("www/favicon/favicon.html"),
+    tags$meta(name="description", content="A interactive web tool for network meta-analysis (NMA) that leverages established analysis routines"),
+    tags$meta(name="keywords", content="MetaInsight, NMA, Network Meta Analysis, App"),
+    tags$meta(property="og:title", content="Meta Insight: V4.1.0"),
+    tags$meta(property="og:description", content="A interactive web tool for network meta-analysis (NMA) that leverages established analysis routines"),
```
Comment (JanionNevill) [inline ui.R:45]: Should be "***An*** Interactive web tool..."

File: ui.R [hunk 2]
```diff
@@ -1,794 +1,797 @@
...
+  dashboardHeader(disable = TRUE),
+  dashboardSidebar(disable = TRUE),
+  dashboardBody(
+  tags$head(
+    # load custom stylesheet
+    # To ensure white background and no horizontal scroll bars on ranking panel
+    tags$link(rel = "stylesheet", type = "text/css", href = "app.css"),
+    includeHTML("www/favicon/favicon.html"),
+    tags$meta(name="description", content="A interactive web tool for network meta-analysis (NMA) that leverages established analysis routines"),
+    tags$meta(name="keywords", content="MetaInsight, NMA, Network Meta Analysis, App"),
```
Comment (JanionNevill) [inline ui.R:43]: do we want/need "meta analysis" as a separate keyword, or does it already match due to it being a subset?

File: www/app.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.content-wrapper {
+  background-color: #fff;
```
Comment (JanionNevill) [inline www/app.css:2]: Is that a standard way of writing a hex colour? Should it be `#ffffff`?

**Review by JanionNevill — CHANGES_REQUESTED** [review]

File: www/app.css (hunk 3 above)
Comment (RyanJField) [reply www/app.css:2]: I didn't write this but it is an accepted way of writing white. See [short notation](https://www.w3.org/TR/css-color-3/#numerical) in the W3C recommendations.

File: ui.R (hunk 2 above)
Comment (RyanJField) [reply ui.R:43]: They should probably be split, but most search engines ignore them anyway, but they are useful to know what keywords you should be targeting.

Comment (RyanJField) [reply ui.R]: I have split these now.

File: ui.R (hunk 1 above)
Comment (RyanJField) [reply ui.R]: Thank you, changed

**Review by JanionNevill — APPROVED** [review]

**Review by nabury — APPROVED** [review]

**Pull request merged**